### ISSUE TITLE
xlsx input/output format problem

### ISSUE THREAD

**Issue by bengfitzpatrick** [kdahlquist/GRNmap#43]
when using an xlsx input file, I get the following error:

Error using copyfile
No matching files were found.

Error in output (line 20)
copyfile([input_file '.xls'], output_file);

Error in GRNmodel (line 19)
GRNstruct = output(GRNstruct);

When using an xls file, this error does not occur.

**Comment by juansc** [comment]
We want to make sure that an .xlsx file gets outputted regardless of what format the input sheet is.

**Comment by kdahlquist** [comment]
If the user inputs .xls, they should get as output .xls.
If they input .xlsx, they shoud get as ouput .xlsx

The bug was that .xlsx files were not working.  So, we need the program to do the above.

**Comment by kdahlquist** [comment]
This is the highest priority for the next two week period from March 12-March 27.

**Comment by juansc** [comment]
This is fixed please review.

**Comment by kdahlquist** [comment]
verified that this works with the beta code as of 2015-03-25 with the following input sheet "Input_21_Gene_Network_Sigmoid_Model_Estimate_WTCIN5.xlsx" that I copied from Natalie from 2014-11-13. The .xlsx file worked and outputted an .xlsx file.

I then saved this same input sheet as an .xls in Excel, called "Input_21_Gene_Network_Sigmoid_Model_Estimate_WTCIN5_totestxlsversion.xls" and ran it again.  The .xls file worked and outputted an .xls file.

Am closing.
